src/Dapr.Actors/Runtime/ConverterUtils.cs

```diff
@@ -60,5 +63,69 @@ public static string ConvertTimeSpanValueInDaprFormat(TimeSpan? value)
 
             return stringValue;
         }
+
+        public static string ConvertTimeSpanValueInISO8601Format(TimeSpan value, int repetitions)
```

Comment:
Is it possible for someone to pass in a TimeSpan with a repetition and one as a method input? If so, which one wins? Or do we error?

Reply:
I am sorry, did not get your comment here